picard/ui/coverartbox.py

```diff
@@ -33,7 +33,7 @@ class ActiveLabel(QtGui.QLabel):
     """Clickable QLabel."""
 
     clicked = QtCore.pyqtSignal()
-    imageDropped = QtCore.pyqtSignal(QtCore.QUrl)
+    imageDropped = QtCore.pyqtSignal(QtCore.QUrl, QtCore.QByteArray)
```

Comment:
Yes, this seems to have gone under the radar since we recently solidified our Contribution Guide.

Look at https://github.com/metabrainz/picard/blob/master/picard/tagger.py#L95

and https://github.com/metabrainz/picard/blob/master/picard/plugin.py#L220

We should switch this over to snake case too for uniformity :)